### DISCUSSION TITLE
Welcome to wave-ui Discussions!

### DISCUSSION THREAD

**Discussion by antoniandre** [antoniandre/wave-ui#129] [Announcements]
## 👋  Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by cmirea4all** [comment]
Hello, trying to use class attribute on w-input/w-button will trigger vue warn in console. 
Example: <w-input class="mb4" label="Label"></w-input> 

On console it will trigger: 
main.js:14 [Vue warn]: Wrong type passed as event handler to onClass - did you forget @ or : in front of your prop?
Expected function or array of functions, received type string. 
  at <WInput class="mb4" label="Label" > ....

Is there any way to prevent this?
